modules/ecs/main.tf

```diff
@@ -122,11 +124,6 @@ EOF
   tags = var.tags
 }
 
-resource "aws_iam_role_policy_attachment" "policy-attach" {
```

Comment:
We may want to use a for_each here so that users can still use policy attachments against the role outside of this module, especially if we are now exporting the role_arn as an output.

Reply:
Do you want me to try to update to use for_each and not use managed_policies? I can give a stab at it, but my terraform fu is rusty.

Reply:
Ok my terraform fu wasn't so good but my google fu is, so I think I got this the way you requested

Reply:
Any chance this PR could be merged soon? Its holding up a PR on my end 😁